README.md

```diff
@@ -9,6 +9,10 @@ Currently, swaps are only offered in one direction with the `swap` CLI on the bu
 We are working on implementing a protocol where XMR moves first, but are currently blocked by advances on Monero itself.
 You can read [this blogpost](https://comit.network/blog/2021/07/02/transaction-presigning) for more information.
 
+## Maintenance
+
+This repository is currently maintained by community volunteers. While it remains safe to use, the original developers (@comit-network) have stopped contributing. For a more actively maintained version that includes a GUI, please check out this community fork: https://github.com/UnstoppableSwap/core
```

Comment:
i think it’s time to archive this repo and point to https://github.com/UnstoppableSwap/core as the canonical repo.

Reply:
You're probably right. I didn't want to overstep.

Would you be interested in being added as a maintainer to UnstoppableSwap/core? Even if you don't want to contribute as actively, it wouldn't hurt for such an important project to have another pair of eyes looking over what's going on. Even just to give a sense of trust to the community, that none of the contributors turn rogue.

Reply:
sounds good, happy to be added over there as well. although it’s probably apparent i haven’t had much capacity to contribute in a meaningful way

Reply:
and probably doesn’t help that i stopped using matrix because of the spam.

Reply:
I have given you write access on the core project.

It was always nice to work with you. Even if you don't contribute much, we'd love for you to be involved :)

Reply:
> and probably doesn’t help that i stopped using matrix because of the spam.

We have our own Matrix space here: https://matrix.to/#/#unstoppableswap-space:matrix.org

No spam yet and we intend to ban all spammers in the future.